packages/frontend/src/features/notifications/components/DashboardCommentsNotifications.tsx

```diff
@@ -0,0 +1,65 @@
+import { Notification } from '@lightdash/common';
+import { Menu, useMantineTheme } from '@mantine/core';
+import { IconCircleFilled } from '@tabler/icons-react';
+import { FC, useCallback } from 'react';
+import { useHistory } from 'react-router-dom';
+import MantineIcon from '../../../components/common/MantineIcon';
+import { useUpdateNotification } from '../hooks/useNotifications';
+
+type Props = {
+    projectUuid: string;
+    notifications: Notification[] | undefined;
```

Comment:
Can you remove `undefined` from the union here, and forcing the callers to handle an empty array default instead? That would help simplify some of the code in the component itself, and make it less ambiguous.